src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFSubmissionCommand.groovy

```diff
@@ -26,84 +26,108 @@ class LSFSubmissionCommand extends SubmissionCommand {
     LSFSubmissionCommand(
             BatchEuphoriaJobManager parentJobManager,
             BEJob job,
-            String jobName,
+            EscapableString jobName,
             List<ProcessingParameters> processingParameters,
-            Map<String, String> environmentVariables,
-            List<String> dependencyIDs,
-            String command) {
-        super(parentJobManager, job, jobName, processingParameters, environmentVariables, dependencyIDs, command)
+            Map<String, EscapableString> environmentVariables) {
+        super(parentJobManager, job, jobName, processingParameters, environmentVariables)
     }
 
     @Override
-    protected String getJobNameParameter() {
-        return "-J ${jobName}" as String
+    String getSubmissionExecutableName() {
+        return "bsub"
     }
 
     @Override
-    protected String getHoldParameter() {
-        return "-H"
+    protected EscapableString getJobNameParameter() {
+        u("-J ") + jobName
     }
 
     @Override
-    protected String getAccountNameParameter() {
-        return job.accountingName != null ? "-P \"${job.accountingName}\"" : ""
+    protected EscapableString getHoldParameter() {
+        u("-H")
     }
 
     @Override
-    protected String getWorkingDirectoryParameter() {
-        return "-cwd ${job.workingDirectory ?: WORKING_DIRECTORY_DEFAULT}" as String
+    protected EscapableString getAccountNameParameter() {
+        job.accountingName != null ?
+            u("-P ") + job.accountingName :
+            u("")
     }
 
     @Override
-    protected String getLoggingParameter(JobLog jobLog) {
-        return getLoggingParameters(jobLog)
+    protected EscapableString getWorkingDirectoryParameter() {
+        ConcatenatedString result = c(u("-cwd "))
+        if (job.workingDirectory) {
```

Comment:
other than the `c(u("-cwd "))` vs `c(u("--chdir "))` LSF and Slurm methods do the same. Might be worth to move somewhere else and combine it